src/Core/ELILO.pm

```diff
@@ -172,6 +172,7 @@ sub GetMetaData() {
 	noedd30		=> "bool:Don't force EDD30 Mode if not set:",
 	fpswa		=> "path:Specify the Filename for a specific FPSWA to load:",
 	"secure-boot"	=> "string:Support Secure Boot:auto",
+	multipath	=> "int:Number of additional multipath entries:0",
```

Comment:
hmm, but problem is that it is shown for every user even without mutlipath and also on ia64. let me check code